react/NestedSelect/NestedSelect.jsx

```diff
@@ -114,15 +130,38 @@ class NestedSelect extends Component {
               <Divider />
             </>
           ) : null}
-          {children.map(item => (
-            <ItemRow
-              radioPosition={radioPosition}
-              key={item.key || item.title}
-              item={item}
-              onClick={this.handleClickItem}
-              isSelected={isSelectedWithLevel(item)}
-            />
-          ))}
+          {searchOptions && level === 0 && (
+            <div className="u-mh-1 u-mb-half">
+              <Input
+                placeholder={searchOptions.placeholderSearch}
+                onChange={onChange}
+                value={searchValue}
+              />
+            </div>
+          )}
+
+          {hasSearchResult ? (
```

Comment:
displaySearchResultItem && hasSearchResult ...

Reply:
Done

Reply:
in fact no it is a function that I use and not a component